samples/BikeSharingApp/README.md

```diff
@@ -1,4 +1,100 @@
-# Bike Sharing Sample: Iteratively Develop and Debug Microservices in Kubernetes
+# Deploy the Bike Sharing sample application to Azure Kubernetes Service
 
-## Quickstart
-[Quickstart: Team development on Kubernetes using Azure Dev Spaces](https://docs.microsoft.com/en-us/azure/dev-spaces/quickstart-team-development).
\ No newline at end of file
+BikeSharing is a microservices-based sample application that helps showcase the sandboxing capbilities of [Azure Dev Spaces](https://aka.ms/devspaces). 
+
+Follow the steps below to deploy this sample app to Azure Kubernete Service (AKS).
+
+## Prerequisites
+* [Azure subscription](https://azure.microsoft.com/free)
+* [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [Helm 2.13 (or greater)](https://github.com/helm/helm/blob/master/docs/install.md)
+
+## Configure Azure resources
+
+1. **Create an Azure Kubernetes Service cluster.** You must create an AKS cluster in a [supported region](https://docs.microsoft.com/azure/dev-spaces/about#supported-regions-and-configurations). The below commands create a resource group called `bikesharing-group` and an AKS cluster called `bikesharing-cluster` in the `eastus` region.
+
+    ```bash
+    GROUP_NAME=bikesharing-group
+    AKS_NAME=bikesharing-cluster
+    LOCATION=eastus
```

Comment:
I expect a fair portion of the persons testing this to be on Windows, where these commands aren't going to work (at least not on the standard Command Prompt/PowerShell).
Maybe we could leave the group/cluster names inline, so that people don't get an error when running this?

Reply:
Agree we don't gain much by parametrizing this - there were originally more commands so there was greater benefit. I've updated to leave the names inline.